scala-generator/src/main/scala/models/generator/mock/MockClientGenerator.scala

```diff
@@ -73,48 +93,49 @@ class MockClientGenerator(
     )
   }
 
+  val clientCode =
```

Comment:
Make this a def, if it's going to be overridden.

Reply:
👍 done